collector/lib/CollectorConfig.cpp

```diff
@@ -229,10 +234,8 @@ bool CollectorConfig::UseChiselCache() const {
 }
 
 bool CollectorConfig::UseEbpf() const {
```

Comment:
Given the expansion of the meaning of this method it might be better renamed to `UseBPF` WDYT?

Reply:
Well BPF and eBPF are kind of interchangeable at this point, I know we used `core_bpf` as the collection method but I've also used `CO.RE eBPF` in a bunch of logging messages, I think it's fine to keep this as `UseEbpf`. I think it's more important to understand that this function returns true if ANY kind of eBPF/BPF probe is used, which I think it does.